pkg/sources/adapter/azureiothubsource/adapter.go

```diff
@@ -78,7 +77,7 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 		env.ConnectionString,
 	)
 	if err != nil {
-		log.Fatal(err)
+		logger.Fatal("Failed to obtain IoT client", zap.Error(err))
```

Comment:
This needed to be `Fatalw` to leverage the additional field 😉 
No big deal, maybe we'll notice it in the future and fix the format as part of a larger change.

Reply:
Oopsie. I'll fix that as a part of output format standardization PR.